vowpalwabbit/cb_adf.cc

```diff
@@ -188,6 +188,8 @@ void output_example(vw& all, cb_adf& c, example& ec, v_array<example*>* ec_seq)
   float loss = 0.;
 
   uint32_t action = ec.pred.a_s[0].action;
+  ec.pred.a_s.delete_v();
```

Comment:
@JohnLangford is this OK to do here?  I think it is cause we don't need the `a_s` anymore in this case just the `action`.